README.md

```diff
@@ -140,3 +140,31 @@ If you send the header incorrectly it will respond with 422 status code. If your
   ],
 }
 ```
+
+#### Get tag Timeline
+
+You can get the tag Timeline by making a GET request to route `/hashtag/:hashtag` with authentication header, where `:hashtag` is the tag you want to see the Timeline.
+
+If your token is invalid returns 401. If anything breaks internally returns 500. Otherwise returns 200 with a object like
+
+```js
+{
+  hashtag,
+  tagPosts: [
+    {
+      id, // int, post id
+      name, // string, post owner
+      pic_url, // url, profile picture
+      content, // string, post description
+      link_url, // url, link published
+      link_title, // string, title of the link
+      link_image, // url, link preview
+      link_description, // string
+    },
```

Comment:
I think this could also return the boolean indicating if the user is owner of the post. That way the user can edit/delete the post looking on the hashtag mention too intead of just in timeline.